packages/overlay/src/active-overlay.ts

```diff
@@ -401,10 +402,9 @@ export class ActiveOverlay extends LitElement {
     }
 
     public renderTheme(content: TemplateResult): TemplateResult {
-        const color = this.color as Color;
-        const scale = this.scale as Scale;
+        const { color, scale } = this;
         return html`
-            <sp-theme .color=${color} .scale=${scale}>
```

Comment:
what did the `.` characters you removed here do?

Reply:
Using `.color=${color}` applies the value `color` to `sp-theme` as a property setter, e.g. `document.querySelector('sp-theme').color = color`. If we did this without the `.` or `color=${color}` would apply it as an attribute, e.g. `document.querySelector('sp-theme').setAttribute('color', color)`. The solution used here `color=${ifDefined(color)}` essentially does `color && document.querySelector('sp-theme').setAttribute('color', color)` so that the value is only passed when `typeof color !== 'undefined'`.